### PR TITLE
anomalies will try to not spawn on lava / plasma rivers

### LINKED ISSUE #61634: Anomalies can appear in "Icemoon Wastes" on icebox and become completely impossible to find.
## Testmerges:
The following pull requests are currently test merged:
#61618: '_machinery.dm hotfix' by Ghilker at commit 060e094a0a
#61400: 'Adds a second Detective slot.' by Iamgoofball at commit 85f60a731c
#61467: 'Disable lobby music until sound preferences are re-added to the top bar menu.' by MrStonedOne at commit 6cb5dd1fee
#61509: '[TM Candidate] Overhauls orbit and POI code to fix part of issue #61508 where players could observe /mob/living/new_player on the lobby.' by Timberpoes at commit 3ffb55860d

## Reproduction:
https://puu.sh/Id8Pn/efee7f47fb.png

This happened twice this shift but  I was only dead to see one of them.

### LINKED ISSUE #69382: Flux anomaly takes a plasma sip
Reporting client version: 514.1578

## Round ID:
[189028](https://scrubby.melonmesa.com/round/189028)

## Testmerges:
- [69342](https://www.github.com/tgstation/tgstation/pull/69342) [69181](https://www.github.com/tgstation/tgstation/pull/69181) [69023](https://www.github.com/tgstation/tgstation/pull/69023) [69091](https://www.github.com/tgstation/tgstation/pull/69091)

## Reproduction:
image
Don't think flux anomalies (or maybe potentially all anomalies?) Are supposed to use the river as a spawning location.

### PR DESCRIPTION
**Pull request by ShizCalev**
Fixes #69382
Fixes #61634
:cl: ShizCalev
fix: Anomalies will no longer spawn ontop of lava / plasma rivers.
/:cl:

### CODE REVIEW

File: code/modules/events/anomaly.dm [hunk 1]
```diff
@@ -40,15 +40,15 @@
 	impact_area = findEventArea()
```
Comment (CapybaraExtravagante) [inline code/modules/events/anomaly.dm:40]: Shouldn't you be editing this function instead?

**Comment by Mooshimi** [comment]
Could you make it so it can't be in the ice wastes as well while you're here?

**Comment by newfren** [comment]
https://github.com/tgstation/tgstation/issues/61634 is another similar issue

File: code/modules/events/anomaly.dm [hunk 2]
```diff
@@ -26,6 +26,7 @@
 		/area/station/holodeck,
 		/area/shuttle,
 		/area/station/maintenance,
+		/area/icemoon,
```
Comment (Mooshimi) [inline code/modules/events/anomaly.dm:29]: ❤️

**Pull request merged**